palindromeRecursive.py

```diff
@@ -0,0 +1,36 @@
+theinput = input("What is your text? ")
+
+theinput = theinput.lower()
+
+mywords = theinput.split(" ")
+
+myletters = []
+
+isPalindrome = False
+
+for word in mywords:
+    for letter in word:
+        if letter.isalpha():
+            myletters.append(letter)
+        else:
+            pass
+
+def palindromeTest(letters,idx,idx2):
+    if letters[idx] == letters[idx2]:
+        isPalindrome = True
+        idx += 1
+        idx2 -= 1
+        if idx > idx2:
+            return isPalindrome
+        else:
+            return palindromeTest(letters,idx,idx2)
+    else:
+        isPalindrome = False
+        return isPalindrome
```

Comment:
This function works great! One other option that might be less code is slicing the text each time, testing if the first and last characters are equal, so you don't have to track indexes.